aboutcitus/what_is_citus.rst

```diff
@@ -10,21 +10,35 @@ Citus extends the underlying database rather than forking it, which gives develo
 When to Use Citus
 -----------------
 
-Citus provides users real-time responsiveness over large datasets, most commonly seen in rapidly growing event systems or with time series data. Example use cases include:
+There are two situations where Citus particularly excels: real-time analytics and multi-tenant applications.
+
+Real-Time Analytics
+~~~~~~~~~~~~~~~~~~~
+
+Citus supports real-time queries over large datasets. Commonly these queries occur in rapidly growing event systems or systems with time series data. Example use cases include:
 
 * Analytic dashboards with subsecond response times
 * Exploratory queries on unfolding events
 * Large dataset archival and reporting
 * Analyzing sessions with funnel, segmentation, and cohort queries
 
-For concrete examples check out our customer `use cases <https://www.citusdata.com/solutions/case-studies>`_. Typical Citus workloads involve ingesting large volumes of data and running analytic queries on that data in real-time.
+Citus' benefits here are its ability to parallelize query execution and scale linearly with the number of worker databases in a cluster.
+
+For concrete examples check out our customer `use cases <https://www.citusdata.com/solutions/case-studies>`_.
+
+Multi-Tenant Applications
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another Citus use case is managing the data for multi-tenant applications. These are applications where a single database cluster serves multiple tenants (typically companies), each of whose data is private from the other tenants.
+
```

Comment:
It would be useful to chat with Samay/Sai to get list of possible use-cases. I don't want to block this review on that, but let's follow up by adding an issue in the repo to do so?